src/Providers/DataGrid/Wijmo/Helper/Constants.ts

```diff
@@ -0,0 +1,18 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace Providers.DataGrid.Wijmo.Helper.Constants {
+    export enum CssClasses {
+        CellClass = 'div.dg-cell',
+        ColumnGroup = 'wj-colgroup',
+        Tooltip = 'wj-tooltip',
+        TooltipErrorValidation = 'errorValidation'
+    }
+
+    export enum HTMLAttributes {
+        Class = 'class'
+    }
+
+    export enum HTMLEvent {
+        MouseOver = 'mouseover',
+        MouseOut = 'mouseout'
+    }
```

Comment:
This should be located in another namespace? Something like `OSFramework.DataGrid.Helper.GlobalEnum`?

Reply:
Done! Created the GlobalEnum file and moved this constant there